stratum/hal/lib/bcm/bcm_chassis_manager.cc

```diff
@@ -1981,6 +2001,8 @@ ::util::Status BcmChassisManager::SetPortOptionsForPortGroup(
     // We first get the front panel port info from PHAL. Then using this info
     // (read and parsed from the transceiver module EEPROM) we configure serdes
     // for all BCM ports.
+    // TODO(max): Segfault on shared pointer in GetSfpDataSource() call
+    /*
```

Comment:
I enabled this code path and while it does not crash, it does also not work. For this we need a real serdes config DB file, else the configuration will always fail and thus block the ports.
Even when we have config file, the bcm_sdk_wrapper does not implement serdes config at the moment. Looking at the SDKLT docs, I could only find a single reference to SerDes in `PC_PM_RX_LANE_PROFILE`. Maybe that's something SDK6 specific.

For now we should just correct the comment and continue to ignore the code. There is a stratum_bcm binary (`stratum_bcm_serdes_config_enabled`) on the AS7712, if you want to try this yourself.

Reply:
Oops, please ignore my comment, I thought this is the one for transceiver event.
Please modify the comment since there is no issue with `GetSfpDataSource`